src/FrigadeCarousel/styled.ts

```diff
@@ -0,0 +1,139 @@
+import styled, { css, keyframes } from 'styled-components'
+
+const defaultBorder = css`
+  border: 1px solid #e6e6e6;
+`
+
+const defaultShadow = css`
+  box-shadow: 0 4px 20px 0 rgba(0, 0, 0, 0.06);
+`
+
+const fadeIn = keyframes`
+  from {
+    opacity: 0;
+  } to {
+    opacity: 1;
+  }
+`
+
+const fadeOut = keyframes`
+  from {
+    opacity: 1;
+  } to {
+    opacity: 0;
+  }
+`
+
+export const CarouselScroll = styled.div`
+  margin: 0 -20px;
+  overflow-x: auto;
+  padding-left: 20px;
+  padding-right: 20px;
+  scroll-snap-type: x mandatory;
+
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 0 16px;
+
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+
+  ::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+export const CarouselScrollGroup = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 0 16px;
+  scroll-snap-align: center;
+  scroll-snap-stop: always;
+`
+
+export const StyledCarouselFade = styled.div`
+  animation: ${(props) => (props.reversed ? fadeOut : fadeIn)} 0.25s ease-out;
+  background: linear-gradient(to right, rgba(255, 255, 255, 0), #fff 100%);
+  position: absolute;
+  width: 64px;
+`
+
+export const StyledScrollButton = styled.button`
+  ${defaultBorder}
+  box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.1);
+  align-items: center;
+  border-radius: 50%;
+  background: #fff;
+  display: flex;
+  height: 48px;
+  justify-content: center;
+  position: absolute;
+  width: 48px;
+`
+
+export const Card = styled.div`
+  border-radius: 8px;
+  padding: 20px;
+`
+
+export const StyledCarouselCard = styled(Card)`
+  ${defaultBorder}
+
+  &:active {
+    background: #fafdff;
+  }
+  &:hover {
+    border: 1px solid #0b93ff;
+    cursor: pointer;
+  }
+`
+
+export const StyledCarouselCardImage = styled.img`
+  border-radius: 20px;
+  height: 40px;
+  margin-bottom: 12px;
+  width: 40px;
+`
+
+export const CarouselContainer = styled(Card)`
+  ${defaultShadow}
+`
+
+export const CompletedPill = styled.div`
+  background: #d8fed8;
```

Comment:
You already called it out so just noting for follow-up PR to pass down `appearance` and use `${(props) => props.appearance...`  etc.